components/plugin_marketplace/index.ts

```diff
@@ -7,6 +7,9 @@ import {bindActionCreators, Dispatch, ActionCreatorsMapObject} from 'redux';
 import {GlobalState} from 'mattermost-redux/types/store';
 import {GenericAction, ActionFunc} from 'mattermost-redux/types/actions';
 
+import {setFirstAdminVisitMarketplaceStatus} from 'mattermost-redux/actions/general';
+import {firstAdminVisitMarketplaceStatus as getFirstAdminVisitMarketplaceStatus} from 'mattermost-redux/selectors/entities/general';
```

Comment:
Why is `as getFirstAdminVisitMarketPlaceState` necessary?  Can't the `getFirstAdminVisitMarketPlaceState` function be imported by itself and directly in `mapStateToProps`?

Reply:
It was done since the redux state selector was called `firstAdminVisitMarketPlaceState`, I renamed it to remove the aliasing.